vars/deleteFile.groovy

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/groovy
+
+@NonCPS
+def call(String path) {
```

Comment:
Don't see a better way than having this as global function. Makes no sense to have a helper that.. does whatever.. `t3lib_div_jenkins` ;-) Agree?